pages/common/live-server.md

```diff
@@ -0,0 +1,19 @@
+# live-server
+
+> A simple development http server with live reload capability.
+
+- Serve an index.html file and reload on changes:
+
+`live-server`
+
+- Specify a port from which to serve a file:
+
+`live-server --port={{8081}}`
+
+- Specify a given file to serve:
```

Comment:
Same question here.

Reply:
This is similar to the previous, index.html is the default entry point, this setting allows you to specify a different file from which to start serving.